andela/tesla-ah
# #166241003 User should be able to receive notifications

### what does this PR do?
- Enable user to subscribe for and receive notifications
### description of the task to be completed
- Once a user is opted-in for a notification, depending on a notification type, should be able to receive notifications
- Notifications are of two types: email and in-app notifications
- You receive a notification when: An author you follow publishes a new article, An article you liked has new comments.

### How should it manually tested
- Clone this branch, cd into it
- Run migrations: `yarn migrate`
- Run server `yarn dev`
- There should be at least two users registered
- One should follow another
- If one publishes an article, the system notifies all the followers about the new article
- Bookmark an article for reading later
- If someone else leaves a comment on it, you should be notified

### Pivotal Tracker Story
#166241003
### Use case Doc
https://docs.google.com/document/d/1FD7U_se6mD-pvLkItu2N-FZNWxGy2POpZ1qOIyXzD-g/edit?usp=sharing
